crates/starknet-devnet/tests/common/utils.rs

```diff
@@ -339,6 +340,11 @@ where
     }
 }
 
+/// Shall succeed if `f` fits into u128
```

Comment:
Edwin in our ❤️

Reply:
Actually this comment shalln't be here. It was applicable when I was using `u128` for conversion, but I changed that part in the meantime, so good that you left this comment.

Reply:
Removed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/597/commits/361f70f8fff139eb0a0e20087f3d4ff4eabe5868